api/scpca_portal/models/sample.py

```diff
@@ -56,25 +56,24 @@ def __str__(self):
     def get_from_dict(cls, data, project):
         """Prepares ready for saving sample object."""
 
-        has_multiplexed_data = bool(data.get("multiplexed_with"))
         sample = cls(
             age_at_diagnosis=data["age_at_diagnosis"],
-            demux_cell_count_estimate=(
-                data.get("demux_cell_count_estimate") if has_multiplexed_data else None
-            ),
+            demux_cell_count_estimate=(data.get("demux_cell_count_estimate", None)),
```

Comment:
I believe so. The key is dropped because `demux_cell_count_estimate` is not included in the output metadata file, but we keep the property as an attribute on `Sample` objects. 
- When building `Sample` objects we use `updated_samples_metadata`.
- When combining metadata and writing the metadata files, we filter `updated_samples_metadata` and use  `samples_metadata_filtered_keys`.